### PR TITLE
Update learning_goals.md

### PR DESCRIPTION
**Pull request by Akan186**
---
name: solution review 
about: A template PR for code review with a checklist
---

### Files

- [ ] The file name describes the function's behavior
- [ ] There is a module docstring in the function file
- [ ] The test file's name matches the function file name -
  `/tests/test_file_name.py`
- [ ] There is a module docstring in the tests file

### Unit Tests

- [ ] The test class has a helpful name in PascalCase
- [ ] The test class has a docstring
- Every unit test has
  - [ ] A helpful name
  - [ ] A clear docstring
  - [ ] Only one assertion
  - [ ] There is no logic in the unit test
- [ ] All tests pass
- [ ] There are tests for defensive assertions
- [ ] There are tests for boundary cases

### Function Docstring

- [ ] The function's behavior is described
- The function's arguments are described:
  - [ ] Type
  - [ ] Purpose
  - [ ] Other assumptions (eg. if it's a number, what's the expected range?)
- The return value is described
  - [ ] Type
  - [ ] Other assumptions are documented
- The defensive assertions are documented using `Raises:`
  - [ ] Each assumption about an argument is checked with an assertion
  - [ ] Each assertion checks for _only one_ assumption about the argument
- [ ] Include 3 or more (passing!) doctests

### The Function

- [x] The function's name describes it's behavior
- [x] The function's name matches the file name
- [x] The function has correct type annotations
- [x] The function is not called in the function file

### Do's

- [x] Variable names help to understand the strategy
- [x] Any comments are clear and describe the strategy
- [x] Lines of code are spaced to help show different stages of the strategy

### Don'ts

- [ ] The function's strategy _is not_ described in the documentation
- [ ] Comments explain the _strategy_, **not** the _implementation_
- [ ] The function _does not_ have more comments than code
  - If it does, consider finding a new strategy or a simpler implementation

## Implementation

- [ ] The code passes the formatting checks
- [ ] The code passes all Ruff linting checks
- [ ] The code has no (reasonable) Pylint errors
  - In code review, you can decide when fixing a Pylint error is helpful and
    when it's too restricting.
- [ ] Variables are named with snake_case
- [x] Variable names are clear and helpful
- [ ] The code follows the strategy as simply as possible
- [x] The implementation is as simple as possible given the strategy
- [ ] There are no commented lines of code
- [ ] There are no `print` statements anywhere
- [x] The code includes defensive assertions
- [x] Defensive assertions include as little logic as possible

### CODE REVIEW

**Comment by MyatCharm** [comment]
I will add yours to make process faster

**Comment by Akan186** [comment]
> I will add yours to make process faster

Thank you, but I just now passed the checks. You can merge this now I guess.

File: collaboration/learning_goals.md [hunk 1]
```diff
@@ -28,3 +28,6 @@ documentation, testing, and writing clean, readable code.
 
 MD Jubayer Khan: My goal is to grow as a versatile developer
 by mastering multiple technologies and frameworks.
+
+Thilakan Jegatheeswaran: I aim to contribute to teamwork, create unique code,
```
Comment (MyatCharm) [inline collaboration/learning_goals.md:32]: I think it looks goods now.

**Review by MyatCharm — APPROVED** [review]
